src/utils/index.ts

```diff
@@ -2,16 +2,17 @@ import { BigNumber, ContractInterface, ContractTransaction, ethers, Signer, util
 import { Provider } from '@ethersproject/abstract-provider';
 import ERC20ABI from 'abis/ERC20.json';
 import TokenGeyserABI from 'abis/TokenGeyser.json';
+import { BigNumber as BNJS } from 'bignumber.js';
 
 export function commifyString(number: string): string {
   const parts = number.split('.');
   parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
   return parts.join('.');
 }
 
-export function formatRelays(relays: string | BigInt): string {
-  const relaysByMillion = BigInt(relays) / 1000000n;
-  return relaysByMillion.toString();
+export function formatRelays(relays: string | BNJS): string {
+  const relaysByMillion = new BNJS(relays).div(new BNJS(1000000));
```

Comment:
Didn't we have an utility to make a bignumber in `utils`? if so, let's use that one. :)

Reply:
> Didn't we have an utility to make a bignumber in `utils`? if so, let's use that one. :)

The one in utils uses Javascript's BN.